circle.yml

```diff
@@ -2,7 +2,7 @@ machine:
   environment:
     MAJOR_VERSION_TAG: 1
```

Comment:
I think that can be removed.  Since we went with tool versions in the version string.